### ISSUE TITLE
Error java.lang.ClassNotFoundException when running individual test cases from Android Studio using the plugin

### ISSUE THREAD

**Issue by oh-naoki** [kotest/kotest#3835]
It seems we're encountering an issue with Kotest version 5.8.0 when running individual test cases from Android Studio using the plugin. The exception java.lang.ClassNotFoundException is thrown, and it only occurs in cases where the test case references a data class located in a different directory.

The test case you're running looks like this:

``` kt
class SampleTest : StringSpec({
    "sampletest" { // 👈 Fail
        TestObj(true) shouldBe TestObj(false)
    }

    "sampletest2" { // 👈 Pass
        1 + 2 shouldBe 3
    }
})
```

**Comment by LeoColman** [comment]
Only when running individual tests?
When running, for example, the entire "SampleTest", do all tests behave as expected?

**Comment by nsk90** [comment]
looks it is duplicate of https://github.com/kotest/kotest/issues/3651

**Comment by LeoColman** [comment]
Closed as duplicate of #3651
